CONTRIBUTING.md

```diff
@@ -592,16 +595,22 @@ See PF-1395 for switching this repo to 3.)
 
 **Test user Terra OAuth**
 
-At the beginning of each test, before running terra CLI, [test user is logged in](https://github.com/DataBiosphere/terra-cli/blob/8adf7cdaaa1f74f9407c10cccc8f7c0c4623eb6b/src/test/java/harness/baseclasses/SingleWorkspaceUnit.java#L30).
+At the beginning of each test, before running terra
+CLI, [test user is logged in](https://github.com/DataBiosphere/terra-cli/blob/8adf7cdaaa1f74f9407c10cccc8f7c0c4623eb6b/src/test/java/harness/baseclasses/SingleWorkspaceUnit.java#L30)
+.
 
-[`TestUser.login()` writes `.terra/StoredCredential`](https://github.com/DataBiosphere/terra-cli/blob/8adf7cdaaa1f74f9407c10cccc8f7c0c4623eb6b/src/test/java/harness/TestUser.java#L76), using domain-wide delegation to avoid browser flow.
+[`TestUser.login()` writes `.terra/StoredCredential`](https://github.com/DataBiosphere/terra-cli/blob/8adf7cdaaa1f74f9407c10cccc8f7c0c4623eb6b/src/test/java/harness/TestUser.java#L76)
+, using domain-wide delegation to avoid browser flow.
 
 </td>
 <td valign="top">
 
-**Pet ADC**
+**Test user ADC**
+
+*Nextflow*
+
+Populate `.config/gcloud/application_default_credentials.json`.
```

Comment:
What populates this file?

By the way, are there any responsibilities on test writers or maintainers to clean up the auth at the end of the test run?

Reply:
If you search repo for `config/gcloud/application_default_credentials.json`, you'll find it. I thought about linking to the code directly, but there's a bunch of link I should create to be consistent. I think it's easy enough to search the repo.

[Tests delete `.gcloud/config` before they run.](https://github.com/DataBiosphere/terra-cli/blob/79a938e56f2d95111aeec54175b6d38d9e5deb79/src/test/java/harness/baseclasses/ClearContextUnit.java#L24)